ansys/dpf/core/field_base.py

```diff
@@ -144,25 +155,29 @@ def __del__(self):
             pass
 
     def _connect(self):
-        """Connect to the grpc service"""
+        """Connect to the gRPC service."""
         return field_pb2_grpc.FieldServiceStub(self._server.channel)
     
     def _set_scoping(self, scoping):
-        """
+        """Set the scoping.
+        
         Parameters
         ----------
-        scoping : Scoping
+        scoping : :class"`ansys.dpf.core.scoping.Scoping`
```

Comment:
```suggestion
        scoping : :class:`ansys.dpf.core.scoping.Scoping`
```

Reply:
@PipKat are we supposed to update all the docstrings with this syntax? The hyperlink works well in generated doc as is

Reply:
@cbellot000 We don't have to change it if the current syntax works. I just only have been taught certain ways--so my fault. The function _set_scoping is hidden. I'll have to play around and test one that is visible in the doc.

Reply:
Okay, that's good news @PipKat because we have this syntax everywhere. For each of your commits, the pipeline is ran and the documentation is generated in the artifacts if you want to check your changes